src/woo_publications/publications/admin.py

```diff
@@ -395,3 +397,41 @@ def at_time(self, obj: Document) -> datetime:
     @admin.display(description=_("was associated with"))
     def was_assciated_with(self, obj: Document) -> Organisation | None:
         return obj.publicatie.verantwoordelijke
+
+
+@admin.register(Subject)
+class SubjectAdmin(AdminAuditLogMixin, admin.ModelAdmin):
+    list_display = (
+        "officiele_titel",
+        "publicatiestatus",
+        "registratiedatum",
+        "uuid",
+        "show_actions",
+    )
+    readonly_fields = (
+        "uuid",
+        "registratiedatum",
+        "laatst_gewijzigd_datum",
+    )
+    search_fields = (
+        "uuid",
+        "officiele_titel",
+    )
+    list_filter = (
```

Comment:
Do we want to add the Promoot field as well?

Reply:
let's do it :) probably also good to add it to `list_display`